packages/json-serializer/test/serializeJSON-test.js

```diff
@@ -24,9 +24,15 @@ describe('#serializeJSON', () => {
 
     new namespace.elements.Category([
       new namespace.elements.Category([
-        new namespace.elements.String('hello world', { id: 'message' }),
-      ], { classes: ['dataStructures'] }),
-      element,
+        new namespace.elements.Category([
+          new namespace.elements.Category([
+            element,
+          ], { classes: ['dataStructures'] }),
+        ], { classes: ['api-category'] }),
```

Comment:
```suggestion
        ], { classes: ['api'] }),
```

The class for an api category is just `api` (https://apielements.org/en/latest/element-definitions.html#id35)